english/marshmallow.md

```diff
@@ -955,3 +955,5 @@ Time to [move to Canada](Canada/Canada.md), eh?
 
 
 The crushing weight of all these options causes you to [end your life.](end/end.md)
+
+Before you pulled the plug, someone suggested you find comfort by [visiting jw.org] (visit-jw-dot-org/visit-jw-dot-org.md).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.